config_script.py

```diff
@@ -45,11 +47,42 @@ def convert_object(args: list) -> None:
         sys.exit()
 
 
+def validate_args(parser: argparse.ArgumentParser,
+                  args: argparse.Namespace) -> None:
+    """Validate paths of config file and master dir."""
+    if os.path.isfile(args.config_path) and os.path.isdir(args.master_dir):
+        return
+    parser.print_usage()
+    print(f'{parser.prog}: error: Incorrect file path or directory')
+    sys.exit()
+
+
+def create_parser() -> argparse.ArgumentParser:
+    """Create a parser and optional arguments."""
+    parser = argparse.ArgumentParser(description='Configuration Converter',
+                                     prog='PROG')
+    parser.add_argument('config_path', help='path of fluentd config file')
+    parser.add_argument('master_dir',
+                        help='directory to store master ' + 'config file in')
+    parser.add_argument(
+        '--log_level',
+        default='info',
+        metavar='level',
+        choices=['info', 'fatal', 'error', 'warn', 'debug', 'trace'],
+        help='default: info, other options: fatal,error,warn,debug,trace')
+    parser.add_argument('--log_filepath',
+                        metavar='path',
+                        default='/var/log/ops_agent/ops_agent.log',
+                        help='default: /var/log/ops_agent/ops_agent.log')
+    return parser
+
+
 if __name__ == '__main__':
-    args_passed = sys.argv[1:]
-    get_object(args_passed)
-    fluentd_path = args_passed[0]
-    master_dir = args_passed[1]
-    file_name = os.path.splitext(os.path.basename(fluentd_path))[0]
-    convert_object([master_dir, file_name])
-    subprocess.run(['rm', f'{args_passed[-1]}/config.json'], check=True)
+    parser: argparse.ArgumentParser = create_parser()
+    args: argparse.Namespace = parser.parse_args()
+    validate_args(parser, args)
+    file_name: str = os.path.splitext(os.path.basename(args.config_path))[0]
+    get_object([args.config_path, args.master_dir])
+    convert_object(
+        [args.master_dir, file_name, args.log_level, args.log_filepath])
+    subprocess.run(['rm', f'{args.master_dir}/config.json'], check=True)
```

Comment:
Nit: Here you could use os.path.join().

Reply:
Oh yeah that could work too - I'll change it.